src/Semantics.js

```diff
@@ -451,6 +519,18 @@ Semantics.createSemantics = function(grammar, optSuperSemantics) {
     s.extendOperationOrAttribute.call(s, 'attribute', name, actionDict);
     return proxy;
   };
+  proxy.getOperationNames = function() {
```

Comment:
Hmmm, I wonder if we should just be exposing the operations and attributes themselves...

I'm fine with this for now, just something for us to think about in the future.